tests/phpunit/tests/test-translated-post.php

```diff
@@ -92,6 +92,56 @@ public function test_delete_post_translation() {
 		$this->assertSame( 0, self::$model->post->get_translation( $de, 'fr' ) ); // fails
 	}
 
+	public function test_save_translations() {
+		$posts = self::factory()->post->create_translated(
+			array( 'lang' => 'en' ),
+			array( 'lang' => 'fr' ),
+			array( 'lang' => 'de' )
+		);
+
+		$terms = wp_get_object_terms( $posts, 'post_translations' );
+		$this->assertCount( 1, $terms );
+
+		$this->assertSame( self::$model->post->get_translation( $posts['en'], 'en' ), $posts['en'] );
+		$this->assertSame( self::$model->post->get_translation( $posts['fr'], 'fr' ), $posts['fr'] );
+		$this->assertSame( self::$model->post->get_translation( $posts['de'], 'de' ), $posts['de'] );
+
+		$this->assertSame( self::$model->post->get_translation( $posts['en'], 'fr' ), $posts['fr'] );
+		$this->assertSame( self::$model->post->get_translation( $posts['en'], 'de' ), $posts['de'] );
+
+		$this->assertSame( self::$model->post->get_translation( $posts['fr'], 'en' ), $posts['en'] );
+		$this->assertSame( self::$model->post->get_translation( $posts['fr'], 'de' ), $posts['de'] );
+
+		$this->assertSame( self::$model->post->get_translation( $posts['de'], 'fr' ), $posts['fr'] );
+		$this->assertSame( self::$model->post->get_translation( $posts['de'], 'en' ), $posts['en'] );
+
+		// Removes the translations from the group by updating the German post.
+		self::$model->post->save_translations( $posts['de'], array() );
+
+		$terms = wp_get_object_terms( $posts, 'post_translations' );
+		$this->assertCount( 1, $terms );
+
+		$this->assertSame( self::$model->post->get_translation( $posts['de'], 'fr' ), 0 );
```

Comment:
Isn't the assertion reversed? (same for others)

Reply:
🤔 I don't think so.
We unlink translations from `de` [here](https://github.com/polylang/polylang/blob/8d1269170938b8977dbb4a68be3a4378376fa093/tests/phpunit/tests/test-translated-post.php#L119).
We check everything is good.

Reply:
🥱

Reply:
Done in 744074b55cd2e4dce747c4276873bdde64597313